src/breusch_godfrey.jl

```diff
@@ -69,8 +66,8 @@ function BreuschGodfreyTest{T<:Real}(xmat::AbstractArray{T},e::AbstractVector{T}
 end
 
 testname(::BreuschGodfreyTest) = "Breusch-Godfrey autocorrelation test"
-population_param_of_interest(x::BreuschGodfreyTest) = ("coefficients on lagged residuals up to lag p",
-    "all zero", NaN)
+population_param_of_interest(x::BreuschGodfreyTest) = ("coefficients on lagged residuals
+    up to lag p", "all zero", NaN)
```

Comment:
The spaces and newlines don't get stripped out, so this will be weird. Better to just put the entire line after the `=` on its own line.